clients/cli/src/utils.rs

```diff
@@ -41,6 +41,9 @@ pub fn get_account(db: &Db) -> Account {
                 "No account found, run init, import or help. Error: {:?}",
                 err
             ),
+            Error::InvalidPrivateKey(err) => {
```

Comment:
It's only here because it's in sled, but I don't think it needs to be in sled in the first place